docs/resources/conformance-statement.md

```diff
@@ -328,6 +328,7 @@ We support searching on below attributes and search type.
 | ReferringPhysicianName | X | X | X |
 | StudyDate | X | X | X |
 | StudyDescription | X | X | X |
+| ModalitiesInStudy | X | X | X |
```

Comment:
I have one last question related to the new include field we added.

- We didnt mention anywhere in the documentation, should we do it or is it so obvious?
- Since we support these computed fields, should we return these fields when includefield=all is specified?

Reply:
We only specify the default and all fields, which do not include either of the new computed tags there. We can add it later if customers ask. But I would prefer them to specify explicitly.